test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -3465,53 +3465,6 @@ BOOST_AUTO_TEST_CASE(array_copy_calldata_storage)
 	ABI_CHECK(callContractFunction("retrieve()"), encodeArgs(9, 28, 9, 28, 4, 3, 32));
 }
 
-BOOST_AUTO_TEST_CASE(array_copy_including_mapping)
```

Comment:
Is it possible to change the mappings to a `uint[]` array? I think we might not have many copying tests.

Reply:
I changed the mapping into a `uint[3]` array.